README.md

```diff
@@ -23,7 +24,7 @@ The runtime and target's template are up-to-date with mainstream commit: `master
 
 ## Kotlin Target
 
-The Kotlin target is a plugin for the ANTLR generator.  
+The Kotlin target is a plugin for the ANTLR generator.
```

Comment:
I think the formatter got rid of the double whitespace automatically.

Reply:
I am not sure if you think it should be re-added

Reply:
@ftomassetti I'd add the removed double spaces back (there are two edits for this case). Just for better formatting in the rendered markdown.

Reply:
IDEA seems to remove those spaces on save, making it very inconvenient to keep them. I would instead add an empty line between this sentence and the next one.

Reply:
@ftomassetti it's ok. But FYI, it's caused by the EditorConfig settings. There is trim trailing ws option that is on save (and then you go to commit).

What I do usually is set that option to false, commit the whitespaces, and set it back to true.

Reply:
I think we can consider that in a separate issue, but for now I will merge this (squashing the commit), merge this and then reviewed the pending PRs